java/src/main/java/com/google/budoux/HTMLProcessor.java

```diff
@@ -126,9 +139,9 @@ public void tail(Node node, int depth) {
    * @param html the HTML string to resolve.
    * @return the HTML string of phrases wrapped in non-breaking markup.
    */
-  public static String resolve(List<String> phrases, String html) {
+  public static String resolve(List<String> phrases, String html, String separator) {
```

Comment:
nit: This is a public API, is it ok to change without making `separator` an optional argument?

Reply:
That's a good point! I added the old API for backward compatibility.